aii-core/src/main/perl/Shellfe.pm

```diff
@@ -54,26 +56,26 @@ use constant TIMEOUT    => 60;
 use constant PARTERR_ST => 16;
 use constant COMMANDS   => qw (remove configure install boot rescue firmware livecd status);
 use constant INCLUDE    => 'include';
-use constant NBPDIR     => 'nbpdir';
-use constant CONFIGURE  => 'Configure';
-use constant INSTALLMETHOD      => 'Install';
-use constant BOOTMETHOD => 'Boot';
-use constant REMOVEMETHOD       => 'Unconfigure';
-use constant STATUSMETHOD       => 'Status';
-use constant RESCUEMETHOD       => 'Rescue';
-use constant FIRMWAREMETHOD     => 'Firmware';
-use constant LIVECDMETHOD       => 'Livecd';
 use constant CAFILE     => 'ca_file';
 use constant CADIR      => 'ca_dir';
 use constant KEY        => 'key_file';
 use constant CERT       => 'cert_file';
-
 use constant HWCARDS   => '/hardware/cards/nic';
 use constant DHCPCFG   => "/usr/sbin/aii-dhcp";
 use constant DHCPOPTION => '/system/aii/dhcp/options';
 use constant DHCPPATH  => '/system/aii/dhcp';
 use constant MAC       => '--mac';
 
+# Actual method associated with various actions
+use constant CONFIGURE  => 'Configure';
+use constant INSTALLMETHOD      => 'Install';
```

Comment:
either remove the spaces (like Configure), or add them to CONFIGURe and BOOTMETHOD